report/sections/Chapter 1/Intro.tex

```diff
@@ -1 +1,72 @@
-\chapter{P7 Intro}
+\chapter{Introduction} \label{chap:introduction}
+On the 1st semester of the Software Master’s Degree at \aau{}, the Programming Paradigms course had a problem where most attending students failed the course last year. 
+This resulted in a survey being sent out, and a student task force being assembled to grasp the main flaws with how the course was structured. 
+Here, many students mentioned lacking experience in the programming languages taught in the course given few opportunities to actively train their skills during the semester. 
+
+
+With this in mind, \aau{} chose to restructure the course entirely. 
+The course now focuses on a single programming language instead of multiple. 
+Furthermore, the course features more practical programming instead of following the traditional lecture model usually used at universities. 
+To support this way of teaching, the course lectures have been moved to a more classical classroom setup, but with a monitor at each table and a main monitor at the lecture desk. 
+This is meant to allow students to more easily work together in groups by using the shared monitor.
+In addition, it allows the professor to view and share other groups' monitors such that student solutions can be used for classroom discussions. 
+
+
+This new approach causes its own problems, however. 
+Since students use their own local development environments for solving exercises, other students cannot revisit the discussed exercise solutions as these are not shared, making it impossible to revisit them later. 
+Furthermore, it is difficult to verify whether your own solution is correct and lives up to the requirements of the exercise. 
+In fact, it is not uncommon for people to accidentally misunderstand an exercise and therefore implement an incorrect solution.
+
+\section{Initial Problem} \label{sec:initial-problem}
+The restructuring of the course already seems like a major improvement over the former structure, but as mentioned it still suffers from a few notable problems related to the execution of the exercise sessions:
+\begin{itemize}
+	\item No way to revisit discussed solutions
+	\item No way to verify that your own solution is correct
+	\item No way for the entire group to work together on the same code
+	\item No way for the professor to verify how many exercises the students have solved
+	\begin{itemize}
+		\item Difficult to verify that all students have solved the problems
+		\item Difficult to judge the skill level of the students and whether the difficulty of exercises is suitable
+	\end{itemize}
+\end{itemize}
+
+By summarizing the main problems following the switch to a more practical and exercise focused teaching environment, an initial problem statement can be written.
+\begin{displayquote}
+How can one ensure a mutual feedback loop between students, TAs and the professor during exercise session?
```

Comment:
Det eneste vi rigtigt har beskrevet som et problem (som ikke bare er en liste der påstår problemer) er misforståelse af opgaver og difficulty in verifying solution og finde game solutions frem. Mutual feedback kommer som en overraskelse for mig som læser.